packages/flutter/lib/src/material/dialog_theme.dart

```diff
@@ -38,13 +51,35 @@ class DialogTheme extends Diagnosticable {
   /// Default value for [Dialog.shape].
   final ShapeBorder shape;
 
+  /// Default value for [AlertDialog.titleTextStyle].
+  ///
+  /// Only used in [AlertDialog]s.
```

Comment:
Used to configure the [DefaultTextStyle] for the [AlertDialog.title] widget.

Reply:
Done.